scripts/zkai_attention_kv_stwo_native_seq16_masked_sequence_proof_input.py

```
@@ -0,0 +1,536 @@
+        "token_position": token_position,
+        "query": [((step_index + 2) * (index + 3)) % 9 - 4 for index in range(KEY_WIDTH)],
+        "new_key": [((step_index + 5) * (index + 1) + index) % 7 - 3 for index in range(KEY_WIDTH)],
+        "new_value": [((step_index + 7) * (index + 2) - index) % 11 - 5 for index in range(VALUE_WIDTH)],
+    }
+
+
+def seq16_fixture() -> dict[str, Any]:
+    base = BASE.build_payload()
+    input_steps = list(base["input_steps"])
+    for step_index in range(len(input_steps), SEQUENCE_LENGTH):
+        input_steps.append(extra_step(step_index))
```

Ruff (0.15.12)

[warning] Use `list.extend` to create a transformed list

(PERF401)